clients/kafka-common/src/main/java/org/eclipse/hono/client/kafka/consumer/HonoKafkaConsumer.java

```diff
@@ -111,6 +113,8 @@
      * @param recordHandler The handler to be invoked for each received record.
      * @param consumerConfig The Kafka consumer configuration.
      * @throws NullPointerException if any of the parameters is {@code null}.
+     * @throws IllegalArgumentException if the consumerConfig contains an "enable.auto.commit" property with value
+     *             "true" but no "group.id" property.
```

Comment:
How about using `{@value ConsumerConfig#GROUP_ID_CONFIG}` and `{@value ConsumerConfig#ENABLE_AUTO_COMMIT_CONFIG}` in the _JavaDoc_?

Reply:
@kaniyan I've adapted the Javadocs accordingly.